test/remote/gateways/remote_adyen_test.rb

```diff
@@ -33,6 +33,24 @@ def setup
       :brand => 'elo'
     )
 
```

Comment:
Just a heads up in case you didn't notice -- there appear to be conflicts in this file now.

Reply:
Yep, I'm aware, but I generally hold off on rebasing until I'm all set to go.